sematic/ui/src/components/ScrollingLogView.tsx

```diff
@@ -158,7 +174,7 @@ export default function ScrollingLogView(props: {
           {pullDownTriggerSection}
         </div>
       </Box>
-      {(hasMore && 
+      {(hasMore && !hasError && 
```

Comment:
+1

Reply:
Thanks for the feedback. 

The button is useful when more logs are coming in the future. In the case of exception, the server process is already ceased. It is impossible to produce more logs in the future. At the same time, when pagination is needed to load existing more logs, the `<InfiniteScroll />` will initiate a pull by itself during scrolling, which will happen even before the user sees the button. So there is not much significance to keeping the button.

If there is any misunderstanding, please kindly let me know.

Reply:
Thank you for the explanation!

Reply:
I think the button is less about "there will be more logs in the future" and more about "the UI starts me at the beginning of the logs, but I would like to view the end of them without taking the time to scroll through to the end." So I think it is still useful--the error probably occurred near the end of the logs rather than the beginning, so that is where you will want to start looking at the logs.

Reply:
Looking into the run you are using, it is in an invalid state:
```python
switch_env("user")
    root_id = "e29fa4fef7b946c7bf0ff0e91994880e"
    resolution = api_client.get_resolution(root_id)
    root_run = api_client.get_run(root_id)
    print(f"id: {root_id}")
    print(f"resolution state: {resolution.status}")
    print(f"run state: {root_run.future_state}")
```

Gives:
```
id: e29fa4fef7b946c7bf0ff0e91994880e
resolution state: SCHEDULED
run state: FAILED
```

I'm guessing that the resolution was non-detached, and failed before it could set the resolution state to a terminal one. So it's not a bug with the log reading code. It might be either a legitimate bug in state management logic, or a case of something weird happening while we were developing. Can you maybe try to reproduce in a recent run?

Reply:
> Can you maybe try to reproduce in a recent run?

Regardless, this behavior is orthogonal to the front-end. So just try developing against a run with a valid failure state and we can merge this.